app/factory/Layer.js

```diff
@@ -106,9 +106,77 @@ Ext.define('CpsiMapview.factory.Layer', {
         });
     },
 
+    /**
+     * Creates an OGC WFS layer
+     *
+     * @param  {Object} layerConf The configuration object for this layer
+     * @return {ol.layer.Vector}  WFS layer
+     */
     createWfs: function(layerConf) {
 
-        Ext.log.info('Not implemented yet', layerConf);
+        var url = layerConf.url;
+        if (url.indexOf() == -1) {
+            url+= '?';
+        } else if (!url.endsWith('?') && !url.endsWith('&')) {
+            url += '&';
+        }
```

Comment:
Ther is a utility function in ExtJS that does this, which would probably be even better.